Makefile

```diff
@@ -204,24 +204,22 @@ sync-docs:
 test: test-unit
 test-all: test-unit test-ledger-mock test-race test-cover
 
-test-ledger-mock:
-	@go test -mod=readonly `go list github.com/cosmos/cosmos-sdk/crypto` -tags='cgo ledger test_ledger_mock'
+TEST_PACKAGES=./...
+TEST_TARGETS := test-unit test-unit-amino test-unit-proto test-ledger-mock test-race test-ledger test-race
 
-test-ledger: test-ledger-mock
-	@go test -mod=readonly -v `go list github.com/cosmos/cosmos-sdk/crypto` -tags='cgo ledger'
+test-unit: ARGS=-tags='cgo ledger test_ledger_mock'
```

Comment:
`test-unit` target sets an `ARGS` local variable. But were is a task definition for `test-unit`? We don't specify any dependency here, so it seams that nothing is run.

Reply:
```
alessio@phoenix:~/work/cosmos-sdk$ git log --oneline -n1
60c244dcf (HEAD -> alessio/makefile-test-targets, origin/alessio/makefile-test-targets) Merge branch 'master' into alessio/makefile-test-targets
alessio@phoenix:~/work/cosmos-sdk$ make test-unit
VERSION=goz-phase-1-614-g60c244dcf go test -mod=readonly -tags='cgo ledger test_ledger_mock' ./...
```

Reply:
See https://github.com/cosmos/cosmos-sdk/pull/7290/files#diff-b67911656ef5d18c4ae36cb6741b7965R217

Reply:
Ah, now I understand this. TEST_TARGETS are combine all this commands, and they are listed here only to specify arguments.

Would you mind adding a small comment above TEST_TARGETS please?

Reply:
Done

Reply:
Maybe a bit more:

TEST_TARGETS are constructed by creating local variables for each test target. The rule to for each test target is the same, so they are aggregated in on single dependency: `run-tests`